ci-operator/step-registry/redhat-performance/benchmark-runner/02-operators-deploy/redhat-performance-benchmark-runner-02-operators-deploy-commands.sh

```
@@ -0,0 +1,106 @@
+      -e WORKER_DISK_IDS='${WORKER_DISK_IDS}' \
+      -e WORKER_DISK_PREFIX='${WORKER_DISK_PREFIX}' \
+      -e PROVISION_TIMEOUT='${PROVISION_TIMEOUT}' \
+      -e log_level='INFO' \
+      -v /root/.ssh/id_rsa:/root/.ssh/provision_private_key \
+      -v /root/.kube/config:/root/.kube/config \
+      --privileged '${QUAY_REPO}'"
+  echo "=== Completed: ${operator} ==="
+done
+
+# --- Cleanup container images on provisioner ---
+ssh ${SSH_ARGS} root@"${PROVISION_IP}" "QUAY_REPO=${QUAY_REPO}" bash <<'CLEANUP_EOF'
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)